app/Http/Controllers/DashboardController.php

```diff
@@ -111,9 +114,11 @@ public function getTotalSalesInCur2($allowedCurrencies2)
      */
     public function getTotalSalesInCur1($allowedCurrencies1)
     {
-        $total = Invoice::where('currency', $allowedCurrencies1)
-        ->where('status', '=', 'success')
-        ->pluck('grand_total')->all();
+        $total = Invoice::leftJoin('payments', 'invoices.id', '=', 'payments.invoice_id')
+                 ->where('invoices.status', '=', 'success')
+                 ->orWhere('invoices.status', '=', 'partially paid')
+                 ->where('invoices.currency', $allowedCurrencies1)
+                 ->pluck('payments.amount')->all();
         $grandTotal = array_sum($total);
 
         return $grandTotal;
```

Comment:
getTotalSalesInCur1 and getTotalSalesInCur2 are exactly same and files code dry principle.

Reply:
Fixed